### PR TITLE
Gallery design improvements

### PR DESCRIPTION
**Pull request by niels9001**
- WASDK: window title + Mica 🎉
- WASM: searchbox now works due to a recent Uno upgrade.
- A new design for the titlebar: this brings the searchbox + menu button to the titlebar area, saving us precious vertical space - and inline with W11 design guidelines. Addressing: https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/85

image

@Arlodotexe @michael-hawker the `TitleBar` code is a local copy of the code that lives in Labs: https://github.com/CommunityToolkit/Labs-Windows/pull/459. I'd recommend to flag any non-blocking issues/improvements/feedback there vs. here, as that component would eventually replace this.

### CODE REVIEW

**Review by Mubarrat — APPROVED** [review]

File: CommunityToolkit.App.Shared/Pages/Shell.xaml [hunk 1]
```diff
@@ -17,26 +17,38 @@
...
-        <muxc:NavigationView x:Name="NavView"
-                             Grid.Row="1"
-                             Margin="0,16,0,0"
-                             IsBackButtonVisible="Collapsed"
-                             IsSettingsVisible="False"
-                             ItemInvoked="NavView_ItemInvoked">
-            <Frame x:Name="NavigationFrame"
-                   Navigated="NavigationFrameOnNavigated" />
-            <muxc:NavigationView.AutoSuggestBox>
+                           DisplayMode="Tall"
```
Comment (ghost1372) [inline CommunityToolkit.App.Shared/Pages/Shell.xaml:20]: DisplayMode="Tall", or "Standard", It causes a null error for Window, when running the app.

Comment (ghost1372) [reply CommunityToolkit.App.Shared/Pages/Shell.xaml:19]: Fixed after refactoring👌

**Comment by ghost1372** [comment]
Hi @niels9001
TitleBar is working fine, but How to make our search control look like MS store? 
TitleBar Height is Same, AutoSuggestBox Height also is same, but MS Store AutoSuggestBox has a margin and it is vertically in the center, I used margin, but it got messed up like the picture below. I know that the desired result can be achieved by changing the height of the title bar, but as you can see in the picture, the height of the store title bar and our app is same So how do we achieve it?.
xxx

**Comment by niels9001** [comment]
> Hi @niels9001 TitleBar is working fine, but How to make our search control look like MS store? TitleBar Height is Same, AutoSuggestBox Height also is same, but MS Store AutoSuggestBox has a margin and it is vertically in the center, I used margin, but it got messed up like the picture below. I know that the desired result can be achieved by changing the height of the title bar, but as you can see in the picture, the height of the store title bar and our app is same So how do we achieve it?.
> 
> Update: VerticalAlignment="Center" is not working too.
> 
> xxx

Yeah, needs some more work.. this is a nice test case to see if this is easy to replicate.

I have moved the code to a new `Labs` experiment so we can further iterate and test there with samples included. Let's move this discussion there :-)!

https://github.com/CommunityToolkit/Labs-Windows/pull/459

**Review by Mubarrat — APPROVED** [review]

**Comment by michael-hawker** [comment]
@niels9001 we'll want to do the setup I did for the Converters in #99 for this TitleBar package of labs. Then we'll define the property in the Labs root when we update the tooling submodule reference. In Labs it'll use the source, so that you can work on the titlebar and use it in the sample app, and in Windows it'll just grab the latest NuGet package from the feed (assuming we have the Labs feed setup there).

**Comment by niels9001** [comment]
@michael-hawker since the `TitleBar` code in this PR is just temporarily (later to be swapped out for the control that is in PR for Labs) this should be good to go if nothing is blocking, right?

This would just be a local control for now (with a different namespace) so we shouldn't run into any issues when developing the one in Labs - once that's finished we probably need to follow what you did for the converters?

**Review by Arlodotexe — APPROVED** [review]

**Comment by michael-hawker** [comment]
@niels9001 yeah, should be fine as long as in different namespace. Then once the control is finished, we can swap it out and use the extensions/converters fix the same way.

**Pull request merged**